doc/specs/#5000 - Process Model 2.0/#1032 - Elevation Quality of Life Improvements.md

```diff
@@ -0,0 +1,385 @@
+
+</td>
+</tr>
+<tr>
+<td><strong>Compatibility</strong></td>
+<td>
+
+There are no serious compatibility concerns expected with this changelist. The
+new `elevated` property will be unset by default, so users will heed to opt-in
+to the new auto-elevating behavior.
+
+There is one minor concern regarding introducing the UAC shield on the window.
+We're planning on using themes to configure the appearance of the shield. That
+means we'll need to ship themes before the user will be able to hide the shield
+again.
+
+</td>
+</tr>
+<tr>
+<td><strong>Performance, Power, and Efficiency</strong></td>
+<td>
+
+No changes to our performance are expected as a part of this change.
+
+</td>
+</tr>
+</table>
+
+### Centennial Applications
+
+In the past, we've had a notoriously rough time with the Centennial app
+infrastructure and running the Terminal elevated. Notably, we've had to list all
+our WinRT classes in our SxS manifest so they could be activated using
+unpackaged WinRT while running elevated. Additionally, there are plenty of
+issues running the Terminal in an "over the shoulder" elevation (OTS) scenario.
+
+Specifically, we're concerned with the following scenario:
+  * the current user account has the Terminal installed,
+  * but they aren't an Administrator,
+  * the Administrator account doesn't have the Terminal installed.
+
+In that scenario, the user can run into issues launching the Terminal in an
+elevated context (even after entering the Admin's credentials in the UAC
+prompt).
+
+This spec proposes no new mitigations for dealing with these issues. It may in
+fact make them more prevalant, by making elevated contexts more easily
+accessible.
+
+Unfortunately, these issues are OS bugs that are largely out of our own control.
+We will continue to apply pressure to the centennial app team internally as we
+encounter these issues. They are are team best equipped to resolve these issues.
+
+### Default Terminal & auto-elevation
+
+In the future, when we supporting setting the Terminal as the "default terminal
+emulator" on Windows. When that lands, we will use the `profiles.defaults`
+settings to create the tab we'll hosting the commandline client. If the user has
+`"elevated": true` in their `profiles.defaults`, we'd usually try to
+auto-elevate the profile. In this scenario, however, we can't do that. The
+Terminal is being invoked on behalf of the client app launching, instead of the
+Terminal invoking the client application.
+
+## Future considerations
+
+* If we wanted to go even further down the visual differentiation route, we
+  could consider allowing the user to set an entirely different theme ([#3327])
+  based on the elevation state. Something like `elevatedTheme`, to pick another
+  theme from the set of themes. This would allow them to force elevated windows
+  to have a red titlebar, for example.
+
+## Resources
+
+<!-- Footnotes -->
+
+[#1939]: https://github.com/microsoft/terminal/issues/1939

```

Comment:
Somehow none of these show up in the visual document view at https://github.com/microsoft/terminal/blob/dev/migrie/s/1032-elevation-qol/doc/specs/%235000%20-%20Process%20Model%202.0/%231032%20-%20Elevation%20Quality%20of%20Life%20Improvements.md

Reply:
They aren't supposed to - they just let you put `[#number]` in the markdown and have it get linked up. Otherwise you'd have to put the full URL each time. Usually there are other docs, pages that I put in this section for "citations" essentially, but I didn't do that for this doc apparently.

Reply:
Oh. OK.